frontend/src/components/GrantAccess.vue

```diff
@@ -35,25 +38,25 @@ async function grantAccess() {
   const toast = useToast();
   const grantAccessRequest = toRequest(formData.value)
```

Comment:
I'd rename this variable as we are now using a userRoleAssignment(Create) object, not a grant access request.

Reply:
Variable is renamed. Could you see if it is good? Thank you Basil.

Reply:
Good!